db/migrate/20230120094018_rename_personal_groups_to_member_name.rb

```diff
@@ -0,0 +1,9 @@
+# Update the names of all personal groups to the full name of their first member
+class RenamePersonalGroupsToMemberName < ActiveRecord::Migration[7.0]
+  def change
+    Group.where(tag: :personal_group).each do |group|
+      group.name = group.memberships.first.user.full_name
+      group.save!
+    end
```

Comment:
Improvement idea to implement a more robust migration:

- use up/down methods instead of change
- keep change body as up
- add down body, which reverts this change

It is IMO a good style if migrations are properly reversable, where this is easily possible, as it is the case here.